be/src/storage/storage_engine.cpp

```diff
@@ -714,9 +714,37 @@ Status StorageEngine::_start_trash_sweep(double* usage) {
     // clean unused rowset metas in KVStore
     _clean_unused_rowset_metas();
 
+    _do_manual_compact();
+
     return res;
 }
 
+void StorageEngine::_do_manual_compact() {
+    auto data_dirs = get_stores();
+    for (auto data_dir : data_dirs) {
+        uint64_t live_sst_files_size_before = 0;
+        if (!data_dir->get_meta()->get_live_sst_files_size(&live_sst_files_size_before)) {
+            LOG(WARNING) << "get_live_sst_files_size failed";
```

Comment:
I suggest adding data_dir info to the log, the others are the same

Reply:
you mean put this get_dir_info()
https://github.com/StarRocks/starrocks/blob/87ff3f5b016ae137c738724e68461cedd09ff19c/be/src/storage/data_dir.h#L67-L76
into this warning log?
![image](https://user-images.githubusercontent.com/16617323/161888683-042c3e4e-7b42-45ac-98c8-79c21659ef9d.png)
it seems that do not state the reason for the failure.

Reply:
I'll add info to state which data dir failed.

Reply:
changed. @meegoo